config/default.json

```diff
@@ -7,6 +7,9 @@
     "interval": 300,
     "maxDeviation": 300
   },
+  "stats": {
+    "interval": 10
```

Comment:
How bad would it be if we lowered it even further? To get more up to date data

Reply:
this won't be an issue for the probe, but I don't know how well the server would handle it. I guess we gotta check and see.